<ui version="4.0" >
 <class>PrefAdvanced</class>
 <widget class="QWidget" name="PrefAdvanced" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>466</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="advanced_tab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1" >
      <attribute name="title" >
       <string>&amp;Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="monitor_aspect_icon" >
           <property name="text" >
            <string>icon</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1_8" >
           <property name="text" >
            <string>&amp;Monitor aspect:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>monitoraspect_combo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyComboBox" name="monitoraspect_combo" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>190</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mplayer_use_window_check" >
         <property name="text" >
          <string>&amp;Run MPlayer in its own window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="idx_check" >
         <property name="text" >
          <string>Rebuild &amp;index if needed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="correct_pts_check" >
         <property name="text" >
          <string>C&amp;orrect PTS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="shortnames_check" >
         <property name="text" >
          <string>&amp;Pass short filenames (8+3) to MPlayer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="repaint_video_background_check" >
         <property name="text" >
          <string>Repaint the backgroun&amp;d of the video window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>R&amp;un the following actions every time a file is opened. The actions must be separated with spaces:</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="buddy" >
          <cstring>actions_to_run_edit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="actions_to_run_edit" />
       </item>
       <item>
        <widget class="Line" name="line" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="colorkey_label" >
           <property name="text" >
            <string>&amp;Colorkey:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>colorkey_view</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="colorkey_view" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changeButton" >
           <property name="text" >
            <string>C&amp;hange...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>131</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>489</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2" >
      <attribute name="title" >
       <string>Options for MP&amp;layer</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel3_2" >
         <property name="text" >
          <string>Here you can pass extra options to MPlayer.
Write them separated by spaces.
Example: -flip -nosound</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel1_3" >
           <property name="text" >
            <string>&amp;Options:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>mplayer_args_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_args_edit" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel4_2" >
         <property name="text" >
          <string>You can also pass additional video filters.
Separate them with ",". Do not use spaces!
Example: scale=512:-2,eq2=1.1</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel2_3" >
           <property name="text" >
            <string>V&amp;ideo filters:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>mplayer_vfilters_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_vfilters_edit" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel5_2" >
         <property name="text" >
          <string>And finally audio filters. Same rule as for video filters.
Example: resample=44100:0:0,volnorm</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel3_3" >
           <property name="text" >
            <string>Audio &amp;filters:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>mplayer_afilters_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mplayer_afilters_edit" />
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_proxy" >
      <attribute name="title" >
       <string>&amp;Network</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="internet_connection_groupbox" >
         <property name="title" >
          <string>Network Connection</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QRadioButton" name="ipv4_button" >
            <property name="text" >
             <string>IPv&amp;4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ipv6_button" >
            <property name="text" >
             <string>IPv&amp;6</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Proxy</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>You can set a proxy for internet connections (currently only used for subtitle downloading).</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="use_proxy_check" >
            <property name="text" >
             <string>&amp;Enable proxy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" native="1" name="proxy_container" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>&amp;Host:</string>
               </property>
               <property name="buddy" >
                <cstring>proxy_hostname_edit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2" >
              <widget class="QLineEdit" name="proxy_hostname_edit" />
             </item>
             <item row="0" column="3" >
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>&amp;Port:</string>
               </property>
               <property name="alignment" >
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy" >
                <cstring>proxy_port_spin</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="4" >
              <widget class="QSpinBox" name="proxy_port_spin" >
               <property name="maximum" >
                <number>65535</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QLabel" name="label_4" >
               <property name="text" >
                <string>&amp;Username:</string>
               </property>
               <property name="buddy" >
                <cstring>proxy_username_edit</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="4" >
              <widget class="QLineEdit" name="proxy_username_edit" />
             </item>
             <item row="2" column="0" >
              <widget class="QLabel" name="label_5" >
               <property name="text" >
                <string>Pa&amp;ssword:</string>
               </property>
               <property name="buddy" >
                <cstring>proxy_password_edit</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="4" >
              <widget class="QLineEdit" name="proxy_password_edit" >
               <property name="echoMode" >
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0" >
              <widget class="QLabel" name="label_6" >
               <property name="text" >
                <string>&amp;Type:</string>
               </property>
               <property name="buddy" >
                <cstring>proxy_type_combo</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1" >
              <widget class="QComboBox" name="proxy_type_combo" />
             </item>
             <item row="3" column="2" colspan="3" >
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>261</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>458</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab4" >
      <attribute name="title" >
       <string>Lo&amp;gs</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QCheckBox" name="log_smplayer_check" >
         <property name="text" >
          <string>Log &amp;SMPlayer output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="log_mplayer_check" >
         <property name="text" >
          <string>Log MPlayer &amp;output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="log_mplayer_save_check" >
         <property name="text" >
          <string>A&amp;utosave MPlayer log to file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="log_mplayer_save_name" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line8_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_17" >
         <property name="text" >
          <string>This option is mainly intended for debugging the application.</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel2_10" >
         <property name="text" >
          <string>&amp;Filter for SMPlayer logs:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>log_filter_edit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="log_filter_edit" />
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>advanced_tab</tabstop>
  <tabstop>monitoraspect_combo</tabstop>
  <tabstop>mplayer_use_window_check</tabstop>
  <tabstop>idx_check</tabstop>
  <tabstop>correct_pts_check</tabstop>
  <tabstop>shortnames_check</tabstop>
  <tabstop>repaint_video_background_check</tabstop>
  <tabstop>actions_to_run_edit</tabstop>
  <tabstop>colorkey_view</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>mplayer_args_edit</tabstop>
  <tabstop>mplayer_vfilters_edit</tabstop>
  <tabstop>mplayer_afilters_edit</tabstop>
  <tabstop>ipv4_button</tabstop>
  <tabstop>ipv6_button</tabstop>
  <tabstop>use_proxy_check</tabstop>
  <tabstop>proxy_hostname_edit</tabstop>
  <tabstop>proxy_port_spin</tabstop>
  <tabstop>proxy_username_edit</tabstop>
  <tabstop>proxy_password_edit</tabstop>
  <tabstop>proxy_type_combo</tabstop>
  <tabstop>log_smplayer_check</tabstop>
  <tabstop>log_mplayer_check</tabstop>
  <tabstop>log_mplayer_save_check</tabstop>
  <tabstop>log_mplayer_save_name</tabstop>
  <tabstop>log_filter_edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>log_mplayer_save_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>log_mplayer_save_name</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>249</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_proxy_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxy_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>239</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
